src/RestApi/StoreApi/README.md

```diff
@@ -886,9 +904,18 @@ Example response:
 
 ```json
 {
-	"code": "20off",
-	"total_discount": "1667",
-	"total_discount_tax": "333"
+  "code": "20off",
+  "totals": {
+    "currency_code": "GBP",
+    "currency_symbol": "£",
+    "currency_minor_unit": 2,
+    "currency_decimal_separator": ".",
+    "currency_thousand_separator": ",",
+    "currency_prefix": "£",
+    "currency_suffix": "",
```

Comment:
> I think it'd be better if currency was a top level property in the response.

That would be inconsistent with other endpoints. Using `totals` object with currency context here matches the cart endpoint. Only `totals` need currency in this case.

FWIW https://stripe.com/docs/api/charges/create returns currency code at the same level of amounts it applies to too.

> Also, ideally, all you'd have to return is the currency code and that would be used to inform all other details. So just return a currency_code: 'GBP' property and value. 

This is possible, however, just yesterday you were concerned with multiple requests. I don't think it hurts to return currency information needed for value formatting with responses. Again, all these endpoints use a consistent format, and formatting is not possible with just code unless you're making assumptions about the format, or relying on multiple requests. Also bare in mind _minor_unit_ is very important regardless of formatting.

> Otherwise we're potentially duplicating all this information across all responses involving money values.

Correct, but it may not be duplication. e.g:

- If multiple currencies were supported and we needed to return price objects per currency
- Getting a collection of orders where different currency was used per order. 

Keeping currency with the prices makes that possible. It gives context to the provided values.

So I disagree with your assessment that this data is unnecessary.